addons/mail/static/src/components/activity/activity_tests.js

```diff
@@ -995,6 +987,7 @@ QUnit.test('activity click on cancel', async function (assert) {
             </div>
         `,
     });
+    registerMessagingComponent(ParentComponent);
```

Comment:
It would be nice if it was possible to unregister a component as well then, because this means the global state will be different before and after this test.